katharsis-rs/src/main/java/io/katharsis/rs/JsonApiResponseFilter.java

```diff
@@ -38,6 +40,13 @@ public JsonApiResponseFilter(KatharsisFeature feature) {
 	public void filter(ContainerRequestContext requestContext, ContainerResponseContext responseContext) throws IOException {
 		Object response = responseContext.getEntity();
 		if (response == null) {
+			if (feature.getBoot().isNullDataResponseEnabled()) {
+				Document document = new Document();
+				document.setData(Nullable.nullValue());
+				responseContext.setEntity(document);
```

Comment:
I am looking at this class in master and I don't see this `setEntity` method anywhere. Did I miss something?